Lesson10/src/homework10/Shop.java

```diff
@@ -0,0 +1,24 @@
+package homework10;
+
+import java.util.LinkedList;
+import java.util.List;
+
+public class Shop {
+    private final List<CashRegister> allCashRegisterInShop;
+
+    public Shop() {
+        allCashRegisterInShop = new LinkedList<>();
+    }
+
+    public List<CashRegister> cratingAll() {
+        CashRegister cashRegister = new CashRegister("Tamara", "Dollar", 5000);
+        CashRegister cashRegister1 = new CashRegister("Sveta", "Euro", 3000);
+        CashRegister cashRegister2 = new CashRegister("Galya", "Rub", 1000);
+        CashRegister cashRegister3 = new CashRegister("Tanya", "Rub", 2000);
+        allCashRegisterInShop.add(cashRegister);
+        allCashRegisterInShop.add(cashRegister1);
+        allCashRegisterInShop.add(cashRegister2);
+        allCashRegisterInShop.add(cashRegister3);
+        return allCashRegisterInShop;
```

Comment:
you can just do 
        allCashRegisterInShop.add(new CashRegister("Tamara", "Dollar", 5000));
        allCashRegisterInShop.add(new CashRegister("Sveta", "Euro", 3000));
        
to save some space, because you won't need these variables